src/modules/shared/components/HeaderPixwaySDK/HeaderPixwaySDK.tsx

```diff
@@ -80,12 +80,8 @@ const _HeaderPixwaySDK = ({
     } else setOpenedTabs(!openedTabs);
   };
 
-  const tabsToPass =
-    tabs ||
-    context?.defaultTheme?.header?.styleData?.tabs?.map((l: any) => ({
-      name: l.label,
-      router: l.value,
-    }));
+  const defaultTabs = tabs || context?.defaultTheme?.header?.styleData?.tabs;
+  const tabsToPass = defaultTabs?.map(mapOptionsToTabs);
```

Comment:
Eu acho que isso vai dar problema.... quando tabs vier como props sem ser do constructor isso não vai funcionar pois ele ja vira da forma correta
```suggestion
  const tabsToPass = tabs ? tabs : defaultTabs?.map(mapOptionsToTabs);
```